credentials/settings/base.py

```diff
@@ -556,3 +559,16 @@
 # Plugin Django Apps
 INSTALLED_APPS.extend(get_plugin_apps(PROJECT_TYPE))
 add_plugins(__name__, PROJECT_TYPE, SettingsType.BASE)
+
+# Verifiable credentials settings
+
+# .. toggle_name: ENABLE_VERIFIABLE_CREDENTIALS
+# .. toggle_implementation: DjangoSetting
+# .. toggle_default: False
+# .. toggle_description: Determines if the Credentials IDA uses digital credentials issuance.
+# .. toggle_warning: Requires the Learner Record MFE to be deployed and used in a given environment if toggled to true.
+# .. toggle_life_expectancy: permanent
+# .. toggle_permanent_justification: Digital Credentials are optional for usage.
+# .. toggle_creation_date: 2023-02-02
+ENABLE_VERIFIABLE_CREDENTIALS = False
+VC_DEFAULT_TYPE = "VC"
```

Comment:
This should go into [inner settings](https://youtrack.raccoongang.com/issue/OeX_Cred-310).

Reply:
Hardcode vs constants.

Reply:
Removed in this PR due to `OeX_Cred-310`